glfw/src/Window.zig

```diff
@@ -3221,3 +3221,104 @@ test "setScrollCallback" {
         }
     }).callback);
 }
+
+fn failedToCreateWindow(err: Error) void {
+    // return without fail, because most of our CI environments are headless / we cannot open
+    // windows on them.
+    std.debug.print("note: failed to create window: {}\n", .{err});
+    return;
+}
+
+test "hint-attribute default value parity" {
+    try glfw.init(.{});
+    defer glfw.terminate();
+    const window_a = @as(Error!Window, handle: {
+        // manually create window, to avoid call to `Window.Hints.set` in `Window.create`, which
+        // would overwrite the default hint values set by GLFW itself.
+        const handle = c.glfwCreateWindow(640, 480, "Hello, Zig!", null, null);
+        getError() catch |err| break :handle err;
+        break :handle Window.from(handle.?) catch |err| break :handle err;
+    }) catch |err| return failedToCreateWindow(err);
+    defer window_a.destroy();
+
+    const window_b = Window.create(640, 480, "Hello, Zig!", null, null, .{}) catch |err| return failedToCreateWindow(err);
```

Comment:
All else equal, I'd prefer to keep tests a bit isolated so maybe ditch the `failedToCreateWindow(err)` in this case

Reply:
That's fair. I made it because I was running into some weird issues with block return and 'catching' errors from it, but I eventually figured it out, just didn't think to remove it.